spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper" 
+
+RSpec.describe "Recipe" do 
+  before(:each) do 
+    @salt = Ingredient.create!(name: "Salt", cost: 4)
+    @peas = Ingredient.create!(name: "Peas", cost: 7)
+    @beef = Ingredient.create!(name: "Ground Beef", cost: 24)
+    @pepper = Ingredient.create!(name: "Pepper", cost: 6)
+    @marinara = Ingredient.create!(name: "Marinara", cost: 22)
+
+    @shepherd_pie = Recipe.create!(name: "Shepherd's Pie", complexity: 9, genre: "Meat Lovers")
+
+    @shep_beef = RecipeIngredient.create!(recipe_id: @shepherd_pie.id, ingredient_id: @beef.id)
+    @shep_salt = RecipeIngredient.create!(recipe_id: @shepherd_pie.id, ingredient_id: @salt.id)
+    @shep_peas = RecipeIngredient.create!(recipe_id: @shepherd_pie.id, ingredient_id: @peas.id)
+  end
+
+  it "shows a total cost of all the ingredients in the recipe" do 
+    visit "/recipes/#{@shepherd_pie.id}"
+
+    expect(page).to have_content("Shepherd's Pie")
+    expect(page).to have_content("Ingredients")
+    expect(page).to have_content("Ground Beef")
+    expect(page).to have_content("Peas")
+    expect(page).to have_content("Salt")
+    expect(page).to have_content("Complexity")
+    expect(page).to have_content("9/10")
```

Comment:
These assertions could be made more robust if they were asserted along with their labels, i.e. instead of testing that "9/10" appears somewhere on the page, assert that "Complexity: 9/10" is present so that you know it's in the right place